requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 
 # For running doctests (during "python setup.py test")
 Sphinx >= 1.4.5
+sphinx_bootstrap_theme
```

Comment:
Well, I stand with what I said. If the developer need to run the doctests, them it makes sense to me that ``requirements-docs.txt`` (docs related requirements) are installed, doesn't it?